charts/auto-loans/origination-activity/figure-1a__number.html

```diff
@@ -0,0 +1,13 @@
+    
+<p class="block short-desc">
+    <strong>Source:</strong> <a href="https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv">https://github.com/cfpb/consumer-credit-trends/blob/master/data/num_data_AUT.csv</a><br>
```

Comment:
Can we use the direct raw link here instead, and for all the other source CSV links? Like https://raw.githubusercontent.com/cfpb/consumer-credit-trends/master/data/num_data_AUT.csv, rather than sending people to the GitHub viewing page.